ui/package.json

```diff
@@ -23,7 +23,7 @@
         "build": "rm -rf dist && webpack --config ./src/app/webpack.prod.js",
         "test": "react-scripts test",
         "eject": "react-scripts eject",
-        "protogen": "swagger-codegen generate -i ../pkg/apiclient/rollout/rollout.swagger.json -l typescript-fetch -o src/models/rollout/generated"
+        "protogen": "../hack/swagger-codegen.sh generate -i ../pkg/apiclient/rollout/rollout.swagger.json -l typescript-fetch -o src/models/rollout/generated"
```

Comment:
is this expected change?

Reply:
There was issue with swagger-codegen version. Recent version was causing many unknown errors. So Alex decided to fix the swagger-codegen version to 3.0.25. He has written script swagger-codegen.sh seperately, so we need protogen to point to that script. Thats why we made this change.

Reply:
got it